runners/spark/spark_runner.gradle

```diff
@@ -113,10 +117,6 @@ test {
   }
 
   maxParallelForks 4
-  useJUnit {
```

Comment:
previously these unit tests were excluded during `test` runs because they failed. instead these were run as part of `validatesRunnerStreaming`.
I fixed them so they can be run as normal unit tests